lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -492,14 +492,15 @@ def get_target_from_rule(module, client, rule, name, group, groups, vpc_id):
     target_group_created = False
 
     validate_rule(module, rule)
-    if rule.get('group_id') and re.match(FOREIGN_SECURITY_GROUP_REGEX, rule['group_id']) and current_account_id not in rule['group_id']:
+    if rule.get('group_id') and re.match(FOREIGN_SECURITY_GROUP_REGEX, rule['group_id']):
```

Comment:
Nervous about backporting as this adds new functionality.

Reply:
Oh, testing 2.5, it looks like this was supported at the time as well. There's just a lack of documentation regarding it. Not new at all.